src/three/math/TileBoundingVolume.js

```diff
@@ -73,10 +66,9 @@ export class TileBoundingVolume {
 		if ( obb ) {
 
 			// the obb transform contains no scale
-			_ray.copy( ray ).applyMatrix4( obb.inverseTransform );
-			if ( _ray.intersectBox( obb.box, _obbVec ) ) {
+			if ( obb.intersectsRay( ray, _obbVec ) ) {
 
-				obbDistSq = obb.box.containsPoint( _ray.origin ) ? 0 : _ray.origin.distanceToSquared( _obbVec );
+				obbDistSq = obb.box.containsPoint( ray.origin ) ? 0 : ray.origin.distanceToSquared( _obbVec );
```

Comment:
This this point had the same issue as the function pointed out in #623 - the transform scale wasn't accounted for.

Reply:
Thanks for fixing this issue too. =)